target/snitch_cluster/sw/apps/Makefile

```diff
@@ -22,6 +22,7 @@ SUBDIRS += snax-mac-simple/untiled
 SUBDIRS += snax-mac-simple/tiled
 SUBDIRS += snax-gemm-engine
```

Comment:
@xiaoling-yi  in the new PR, make sure to update this test to use the block GEMM. Basically, it's the same base GEMM test BUT using the block GEMM.